packages/sit-onyx/src/utils/index.ts

```diff
@@ -1,6 +1,6 @@
 /**
- * Checks whether the given link is external (starts with http:// or https://).
+ * Checks whether the given link is internal / a relative link of the current application.
  */
-export const isExternalLink = (href: string): href is `http${"s" | ""}://${string}` => {
-  return /^http(s?):\/\//.test(href);
+export const isInternalLink = (href: string) => {
```

Comment:
What was the reason, again, why we don't check against the actual origin?
E.g.
```ts
const hrefUrl = new URL(href, window.location);
return location.origin === hrefUrl.origin;
```

Reply:
Because we would need to take care about SSR that we use this utility only onMounted

Reply:
Per my understanding, we can just use:
```suggestion
export const isInternalLink = (href: string) => !URL.canParse(); // if it can be parsed the URL is a full URL with some protocol and domain
```

Reply:
Nice! Seems to work fine, I updated it